core/templates/base-components/base-content.component.ts

```diff
@@ -31,13 +31,12 @@ import { SidebarStatusService } from 'services/sidebar-status.service';
 import { BackgroundMaskService } from 'services/stateful/background-mask.service';
 import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
 import { NavigationEnd, Router } from '@angular/router';
-
 import './base-content.component.css';
 
-
 @Component({
   selector: 'oppia-base-content',
-  templateUrl: './base-content.component.html'
+  templateUrl: './base-content.component.html',
+  styleUrls: ['./base-content.component.css']
```

Comment:
Here and everywhere else, we need this so that the styles are included in aot build. The import statements will not work in angular aot. Similarly, the styleUrls are removed by our custom angular template loader webpack plugin.